programs/compressed-token/program/docs/INSTRUCTIONS.md

```
@@ -79,24 +81,26 @@ every instruction description must include the sections:
-4. **Claim** - Rent reclamation from expired compressible accounts
-5. **Close Token Account** - Close decompressed token accounts with rent distribution
-6. **Decompressed Transfer** - SPL-compatible transfers between decompressed accounts
-7. **Withdraw Funding Pool** - Withdraw funds from rent recipient pool
-8. **Create Token Pool** - Create initial token pool PDA for SPL/T22 mint compression
-9. **Add Token Pool** - Add additional token pools for a mint (up to 5 per mint)
-10. **CToken MintTo** - Mint tokens to decompressed CToken account
-11. **CToken Burn** - Burn tokens from decompressed CToken account
-12. **CToken Freeze/Thaw** - Freeze and thaw decompressed CToken accounts
-13. **CToken Approve/Revoke** - Approve and revoke delegate on decompressed CToken accounts
-14. **CToken Checked Operations** - ApproveChecked, MintToChecked, BurnChecked with decimals validation
+## Compressed Token Operations (`compressed_token/`)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)